### DISCUSSION TITLE
Cannot download files from storage

### DISCUSSION THREAD

**Discussion by hrbengtsen** [supabase/supabase#1676] [Questions] [answered]
I could download files just fine a couple days ago, but now all of a sudden i get 400's on all my images. I even went back on vercel, where I deploy and see if old preview deployments could still retrieve the images, but got the same error.

I download them like this:
`const { data, error } = await supabase.storage.from('public').download(path);`

Where I have for example logged path for an image to be: `recipes/Avocado-chicken-pita-153.jpg`

And on supabase the image is there in bucket public in folder recipes:
![Skærmbillede 2021-05-22 104648](https://user-images.githubusercontent.com/31827787/119220520-3a1d1f00-baeb-11eb-8a91-76c21cb47d84.png)

I have also tried re-uploading the same image with the same path, which works as you can see in the screenshot, though still same error when downloading. I have row-level security on, but turning it off doesn't remove the error either.

Very confused as to what is causing this.

**Comment by inian** [comment, marked as answer]
Hey @hrbengtsen can you email beta@supabase.io with your project id (it's in the URL) and I will take a look

**Reply by hrbengtsen** [reply]
I have done that now, thanks

**Reply by inian** [reply]
This is due to https://github.com/supabase/storage-api/issues/22. Temporary fix is to change bucket name to anything other than `public` or `sign`. Proper fix will be pushed out in a couple of days
